src/scripts.js

```diff
@@ -101,18 +75,20 @@ function favoriteCard(event) {
 }
 
 function cardButtonConditionals(event) {
-  getData()
-    .then(allData => {
-      if (event.target.classList.contains('favorite')) {
-        favoriteCard(event);
-      } else if (event.target.classList.contains('card-picture')) {
-        displayDirections(event);
-      } else if (event.target.classList.contains('home')) {
-        favButton.innerHTML = 'View Favorites';
-        let cookbook = new Cookbook(allData.recipeData);
-        populateCards(cookbook.recipes);
-      }
-    })
+  console.log("event ", event);
+  // getData()
+  //   .then(allData => {
+  if (event.target.classList.contains('favorite')) {
+    favoriteCard(event);
+  } else if (event.target.classList.contains('card-picture')) {
+    displayDirections(event);
+  } else if (event.target.classList.contains('home')) {
+    favButton.innerHTML = 'View Favorites';
+    console.log('fjkdhsfkjsdhf')
```

Comment:
remove this console.log

Reply:
work in progress, will remove later of course